DEVELOPMENT.md

```diff
@@ -2,17 +2,25 @@
 
 ## ℹ Token Data
 
-The design system tokens are stored in the tokens.json file. The file consists of token categories and tokens.
+The design system tokens are divided into two types: basic tokens and derived tokens. Basic tokens are stored in the `tokens.json` file. The file consists of token categories and tokens. Derived tokens use the basic tokens to form more complex CSS styles. They are stored in the `derived-tokens.json` file.
 
 ### 📖 Categories
 
 Design tokens are divided into categories based on token type. Categories include a token prefix for the tokens in the category and description information. All tokens belong to exactly one category.
 
-Current categories are:
+Current basic token categories are:
 
 - Colors
 - Typography
 - Spacing
+- Radiuses
+- Transitions
+
+Current derived tokens categories are:
+
+- Gradients
+- Shadows
+- Focuses
```

Comment:
Is `focuses` descriptive enough? Could well be, but technically these are focus indicators I guess. Any thoughts?

Reply:
Fair point. But I think anyone who reads this documentation will understand what "focus" means in this context 🤷

Reply:
I agree. One word is better than multiple in this case as well, so let's leave it as it is.